src/modules/commands/urban.rb

```diff
@@ -17,21 +21,33 @@ def long?
         word   = UrbanDictionary.random.first if term.empty?
         word ||= UrbanDictionary.define(term).first
         next "Couldn't find anything for `#{term}` 😕" unless word
+        url = "... [(Read More)](#{word.url})"
+        word.example.delete!('*')
 
         event.channel.send_embed do |e|
-          e.description = '**Too long to display! Visit the URL by clicking above.**' if word.long?
-          e.add_field name: 'Definition', value: word.text, inline: false unless word.long?
-          e.add_field name: 'Example', value: "*#{word.example}*", inline: false if word.example unless word.long?
+          e.add_field name: 'Definition',
+                      value: word.long_text? ? truncate(word.text, url) : word.text, inline: false
```

Comment:
don't split the lines here, its inconsistent with the rest of the file. one line is fine